src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -89,6 +91,11 @@ public ProcessMetrics(ProcessInstrumentationOptions options)
             description: "Process threads count.");
     }
 
+    public void Dispose()
+    {
+        this.meterInstance.Dispose();
```

Comment:
I'm not sure if it's safe to do this in a scenario where you have multiple Sdks each with an instance of `ProcessMetrics` added since they would be using the same Meter name.

Reply:
I added a test and yes, it does appear to be an issue 😭 Going to disable the test for now. I think multiple provider case is rare so it isn't super urgent that we fix this. But I'm going to spin up an issue and work with @Yun-Ting on it.

Reply:
Opened https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/831